web_refresher/__manifest__.py

```diff
@@ -11,7 +11,17 @@
         "web.assets_backend": [
             "web_refresher/static/src/**/*.scss",
             "web_refresher/static/src/**/*.esm.js",
-            "web_refresher/static/src/**/*.xml",
+            "web_refresher/static/src/xml/refresher.xml",
+            (
```

Comment:
Always explain the whys

```suggestion
            # Load the modification of the master template just after it,
            # for having the modification in all the primary extensions.
            # Example: the project primary view.
            (
```